libpod/kube.go

```diff
@@ -190,56 +203,58 @@ func addContainersToPodObject(containers []v1.Container, podName string) *v1.Pod
 // for a single container.  we "insert" that container description in a pod.
 func simplePodWithV1Container(ctr *Container) (*v1.Pod, error) {
 	var containers []v1.Container
-	result, err := containerToV1Container(ctr)
+	kubeCtr, kubeVols, err := containerToV1Container(ctr)
 	if err != nil {
 		return nil, err
 	}
-	containers = append(containers, result)
-	return addContainersToPodObject(containers, ctr.Name()), nil
+	containers = append(containers, kubeCtr)
+	return addContainersAndVolumesToPodObject(containers, kubeVols, ctr.Name()), nil
 
 }
 
 // containerToV1Container converts information we know about a libpod container
 // to a V1.Container specification.
-func containerToV1Container(c *Container) (v1.Container, error) {
+func containerToV1Container(c *Container) (v1.Container, []v1.Volume, error) {
 	kubeContainer := v1.Container{}
+	kubeVolumes := []v1.Volume{}
 	kubeSec, err := generateKubeSecurityContext(c)
 	if err != nil {
-		return kubeContainer, err
+		return kubeContainer, kubeVolumes, err
 	}
 
 	if len(c.config.Spec.Linux.Devices) > 0 {
 		// TODO Enable when we can support devices and their names
 		devices, err := generateKubeVolumeDeviceFromLinuxDevice(c.Spec().Linux.Devices)
 		if err != nil {
-			return kubeContainer, err
+			return kubeContainer, kubeVolumes, err
 		}
 		kubeContainer.VolumeDevices = devices
-		return kubeContainer, errors.Wrapf(define.ErrNotImplemented, "linux devices")
+		return kubeContainer, kubeVolumes, errors.Wrapf(define.ErrNotImplemented, "linux devices")
```

Comment:
Wait, what's going on here? We're getting devices... and then throwing an error that they're not supported?

Reply:
Should this just be a warning?

Reply:
`TODO Enable when we can support devices and their names` I assume the translation isn't clear for libpod devices to kube devices?